cartography/intel/gcp/dns.py

```diff
@@ -76,7 +76,7 @@ def get_dns_zones(dns: Resource, project_id: str, common_job_parameters) -> List
 
 
 @timeit
-def get_dns_rrs(dns: Resource, dns_zones: List[Dict], project_id: str) -> List[Resource]:
+def get_dns_rrs(dns: Resource, zone: list, project_id: str) -> List[Resource]:
```

Comment:
`list` is invalid, it should be `List`. Also, in this case, i think, it's a Dictionary and not a List.